catroid/src/main/res/layout/activity_upload.xml

```diff
@@ -44,65 +44,69 @@
 
         <ImageView
             android:id="@+id/project_image_view"
-            android:layout_width="100dp"
-            android:layout_height="100dp"
+            android:layout_width="72dp"
+            android:layout_height="65dp"
```

Comment:
what's the reason for changing the width and height? _square aspect ratio!_ as mentioned in the description of https://jira.catrob.at/browse/CATROID-385